pipeline_steps/common.groovy

```diff
@@ -537,6 +537,18 @@ void clone_external_repo(String directory, String repo, String ref, String refsp
 }
 
 
+void merge_pr_chain(String directory='./', List pullRequestIds=null){
```

Comment:
My first thought reading this was that it was going to merge a series of PRs publicly in github. It may be useful to add a comment that this is for merging a series of PRs into a local repo for testing and doesn't affect the public PRs.